app/views/questionnaire.py

```diff
@@ -666,13 +668,15 @@ def _get_block_json(current_location, schema, answer_store, metadata):
     return _evaluate_skip_conditions(block_json, current_location, schema, answer_store, metadata)
 
 
-def _get_schema_context(full_routing_path, group_instance, metadata, answer_store, schema):
+def _get_schema_context(full_routing_path, location, metadata, answer_store, schema):
     answer_ids_on_path = get_answer_ids_on_routing_path(schema, full_routing_path)
+    group_instance_id = get_group_instance_id(g.schema, answer_store, location) if location else None
```

Comment:
No `g.schema` please! Schema is passed as an argument in any case

Reply:
Sorry, I had to merge that PR into this branch, must have missed this one